### PR TITLE
Add can_write_network_config system property

### PR DESCRIPTION
**Pull request by rvykydal**
True on boot_iso and live_io (ie in live environments).

### CODE REVIEW

File: pyanaconda/core/configuration/anaconda.py [hunk 1]
```diff
@@ -179,6 +179,16 @@ def can_configure_network(self):
         """Can we configure the network?"""
         return self._is_boot_iso or self._is_live_os
 
+    @property
+    def can_write_network_config(self):
```
Comment (poncovka) [inline pyanaconda/core/configuration/anaconda.py]: Could the rule be called `provides_network_config`? It seems to be similar to for example `provides_resolver_config`. The current name suggests that we can write to the network configuration of the installation system and that is not true.

File: pyanaconda/core/configuration/anaconda.py [hunk 2]
```diff
@@ -179,6 +179,16 @@ def can_configure_network(self):
...
+        """Can we pass network configuration to installed system?
```
Comment (poncovka) [inline pyanaconda/core/configuration/anaconda.py]: I would use `the target system` here.

**Comment by rvykydal** [comment]
The suggestions by @poncovka applied.

**Comment by rvykydal** [comment]
jenkins, test this please

**Pull request merged**